gpg_group_chat/client/client.py

```diff
@@ -43,20 +44,19 @@ def __handle_messages(self):
         socket_list = [sys.stdin, self._socket]
         self.__prompt()
         while True:
-            ready_to_read,ready_to_write,in_error = select.select(socket_list , [], [])
+            ready_to_read, _, _ = select.select(socket_list, [], [])
             for sock in ready_to_read:
                 if sock == self._socket:
                     # incoming message from remote server
                     data = sock.recv(4096)
-                    if not data :
+                    if not data:
                         print('\nDisconnected from chat server')
                         sys.exit()
-                    else :
-                        #print data
+                    else:
+                        # print data
                         sys.stdout.write(data.decode('utf-8'))
                         self.__prompt()
-
-                else :
+                else:
                     # user entered a message
                     msg = sys.stdin.readline()
                     msg = '['+self._nickname+'] ' + msg
```

Comment:
I think that would be better move this protocol implementation to another PR, according with each open issue, this will make ssl/tls implementation easier.